engine/classes/ElggUser.php

```diff
@@ -127,6 +127,8 @@ public function ban($reason = '') {
 		if (!_elgg_services()->events->trigger('ban', 'user', $this)) {
 			return false;
 		}
+		
+		_elgg_services()->events->trigger('disable:after', 'all', $this);
 
 		$this->ban_reason = $reason;
 		$this->banned = 'yes';
```

Comment:
maybe here
`_elgg_services()->events->triggerAfter('ban', 'user', $this)`

Reply:
please check the updated commit